### PR TITLE
Added paging to playground search in run command

### PR DESCRIPTION
**Pull request by esharf**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-23185
fixes: https://github.com/demisto/demisto-sdk/issues/2885

## Description
Added a fix for the playground search so we will review all the playgrounds, not just the first page.

### CODE REVIEW

File: demisto_sdk/commands/run_cmd/runner.py
Comment (dorschw) [inline demisto_sdk/commands/run_cmd/runner.py:1]: please add a unit test for a multi-playground scenario

File: demisto_sdk/commands/run_cmd/runner.py [hunk 1]
```diff
@@ -106,12 +106,16 @@ def run(self):
...
+        def playground_filter(page: int = 0):
+            return {"filter": {"type": [9], "page": page}}
+
+        answer = self.client.search_investigations(filter=playground_filter())
         if answer.total == 0:
             raise RuntimeError("No playgrounds were detected in the environment.")
-        playgrounds = answer.data
-        if len(playgrounds) > 1:
+        elif answer.total == 1:
+            result = answer.data[0].id
```
Comment (DeanArbel) [inline demisto_sdk/commands/run_cmd/runner.py:117]: Did you check this fix?

**Review by DeanArbel — COMMENTED** [review]
Looks good, please make sure to add tests.

File: demisto_sdk/commands/run_cmd/runner.py (hunk 1 above)
Comment (esharf) [reply demisto_sdk/commands/run_cmd/runner.py:117]: I couldn't check the exact case but did the best I could.

File: demisto_sdk/commands/run_cmd/runner.py
Comment (esharf) [reply demisto_sdk/commands/run_cmd/runner.py:1]: done

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**